packages/nodejs/src/abby/createAbby.ts

```diff
@@ -0,0 +1,44 @@
+import { ABConfig, Abby, AbbyConfig, FlagValueString, FlagValueStringToType } from "@tryabby/core";
+import { F } from "ts-toolbelt";
+import { TestStorageService } from "../shared/StorageService";
+
+export function createAbby<
+  FlagName extends string,
+  TestName extends string,
+  Tests extends Record<TestName, ABConfig>,
+  Flags extends Record<FlagName, FlagValueString> = Record<FlagName, FlagValueString>,
+  ConfigType extends AbbyConfig<FlagName, Tests> = AbbyConfig<FlagName, Tests>,
+>(abbyConfig: F.Narrow<AbbyConfig<FlagName, Tests, Flags>>) {
+  const abbyCoreInstance = new Abby<FlagName, TestName, Tests, Flags>(abbyConfig, {
+    get: (key: string) => {
+      return TestStorageService.get(abbyConfig.projectId, key);
+    },
+    set: (key: string, value: any) => {
+      TestStorageService.set(abbyConfig.projectId, key, value);
+    },
+  });
+
+  //load data and initialise the abby Object
+  abbyCoreInstance.loadProjectData();
```

Comment:
this is async, right? what if some1 creates abby using this method and right after that reading some flags? they arent present then?